scapy/layers/bluetooth.py

```diff
@@ -187,10 +192,16 @@ class HCI_PHDR_Hdr(Packet):
 }
 
 
+# https://www.tcpdump.org/linktypes/LINKTYPE_BLUETOOTH_LINUX_MONITOR.html
+class BT_Mon_Pcap_Hdr(Packet):
```

Comment:
What's the difference between this and the one below?
Why is there nothing bound on top of it?

Reply:
> What's the difference between this and the one below?

PCAP format for Bluetooth Monitor header fields is Big Endian and only has two fields because packet length is encoded somewhere else.

Bluetooth Monitor sockets spit length as the first field and the other two fields are Little Endian.

Those two headers represent the same protocol from two different sources.

>  Why is there nothing bound on top of it?

On top of that there will be a lot of packets that I will contribute in the next PR. Regarding this, I will have to bind subsequent layers by opcode to both headers.

The reason to not ship them in this PR is that I wanted to give them their own PR as to provide reasonable testing for them (it will take me some time).

Reply:
> subsequent layers by opcode to both headers

If you're doing that, it would be a good idea to have one of the layer extend the other so that bindings are shared.

Reply:
Nice, done that. Thanks :)